src/main/java/uk/gov/laa/ccms/caab/controller/application/client/ClientEqualOpportunitiesMonitoringDetailsController.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.laa.ccms.caab.controller.application.client;
+
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_DETAILS;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.SessionAttributes;
+import reactor.core.publisher.Mono;
+import uk.gov.laa.ccms.caab.bean.ClientDetails;
+import uk.gov.laa.ccms.caab.bean.validators.client.ClientEqualOpportunitiesMonitoringDetailsValidator;
+import uk.gov.laa.ccms.caab.service.CommonLookupService;
+import uk.gov.laa.ccms.data.model.CommonLookupDetail;
+
+/**
+ * Controller for handling contact client details selection during the new application process.
```

Comment:
Update javadocs in this class, guess it's copied from elsewhere

Reply:
amended